utils/paramfetch/src/lib.rs

```diff
@@ -1,7 +1,6 @@
 // Copyright 2019-2022 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
-use async_std::task;
```

Comment:
Possible to remove `async-std` from all `Cargo.toml` files?

Reply:
Yes, this pr isn't finished yet. there are few more changes. I'll remove from Cargo.toml at the end.